app/services/log_event_searcher.rb

```diff
@@ -4,7 +4,7 @@ class LogEventSearcher
 
   ALLOWED_EVENTS = ["account.create", "account.update",
                     "account_user.create", "account_user.delete",
-                    "user.create"].freeze
+                    "user.create", "user.suspended", "user.unsuspended"].freeze
```

Comment:
I'm guessing this will conflict with #2257 if that goes in first, but we can manage that conflict when it arises.

Reply:
Yes, I was counting on this, and as I keep moving forward in this ticket Im guessing we'll encounter this situation over and over